examples/device-dashboard/Makefile

```diff
@@ -20,27 +20,32 @@ ifeq ($(OS),Windows_NT)   # Windows settings. Assume MinGW compiler. To use VC:
   MAKE += WINDOWS=1 CC=$(CC)
 endif
 
-all: $(PROG)              # Default target. Build and run program
-	$(RUN) ./$(PROG) $(ARGS)
+# Default target. Build and run program
+all: $(PROG)
+	$(RUN) $(PROG) $(ARGS)
```

Comment:
This works in Windows anyway, I guess make does the magic.
I like it this way better, it is more straightaway for the (Win)dozer